src/q2g-ext-listboxDirective.ts

```diff
@@ -406,7 +405,6 @@ class ListboxController extends utils.RootSingleList implements ng.IController,
                     let that1 = this;
                     this.list.obj.on("changeData", function () {
                         that1.collectionAdapter.calcCollections(that.list.collection);
-                        that1.timeout();
```

Comment:
@thomashaenig method not exists can u check this please ? i have removed this now seems still to work.